### ISSUE TITLE
[Multi Actions] Refactor BaseAction types

### ISSUE THREAD

**Issue by flvndvd** [dust-tt/dust#5170]

**Comment by fontanierh** [comment]
@flvndvd  moving to olumpus 🙏

**Comment by flvndvd** [comment]
Will take care of it by the end of the week.

**Comment by PopDaph** [comment]
@flvndvd can we close?
(context: We're trying to cleanup Olympus. If not done since May maybe it is not urgent).

**Comment by spolu** [comment]
This one should still happen for the purpose of having a unified way of routing to different implementations it's a short one

**Comment by PopDaph** [comment]
I agree it's better to have a unified way @spolu, but this card is 4 months old so it was worth the question. 

We all tend to leave in the middle of refactoring, it's not the first time and won't be the last, and they don't all have cards. We need to be better prioritize the full task when we start a refactoring as part of another project.

**Comment by spolu** [comment]
@flvndvd are you still tracking this? 

> Will take care of it by the end of the week.

:)

**Comment by ElPicador** [comment]
too old closing
